dnf-json

```diff
@@ -181,6 +243,25 @@ class Solver():
         }
 
 class DnfJsonRequestHandler(BaseHTTPRequestHandler):
+    """
+    Answers Http requests to depsolve or dump packages.
+    """
+
+    cache_folders = []
```

Comment:
Do we even need this as an instance variable? It might be simpler to just have this variable locally in `do _POST`.

Reply:
No we don't indeed :)

Reply:
Fixed.